### PR TITLE
New mirror

### PR DESCRIPTION
**Pull request by PardesiCat**
Created PR  from ikstream order 

details in

 https://github.com/BlackArch/blackarch-site/issues/175

### CODE REVIEW

File: blackarch-mirrorlist [hunk 1]
```diff
@@ -92,6 +92,9 @@ Server = https://ftp.halifax.rwth-aachen.de/blackarch/$repo/os/$arch
 #Server = http://mirror.yandex.ru/mirrors/blackarch/$repo/os/$arch
 #Server = ftp://mirror.yandex.ru/mirrors/blackarch/$repo/os/$arch
 
+#South Korea
```
Comment (noraj) [inline blackarch-mirrorlist]:
```suggestion
# South Korea
```

**Review by noraj — CHANGES_REQUESTED** [review]

File: blackarch-mirrorlist (hunk 1 above)
Comment (noraj) [reply blackarch-mirrorlist]: If you can just fix the country spacing so it's like for the others. Outside of this LGTM.

Comment (PardesiCat) [reply blackarch-mirrorlist]: oke did

**Review by noraj — APPROVED** [review]
LGTM

**Comment by PardesiCat** [comment]
B U M P

**Comment by noraj** [comment]
@ikstream @noptrix You guys can handle it? I have received order not to merge mirror related stuff because it need testing.

**Pull request closed without merging**